### PR TITLE
Limit_cou_admins_access_to_org_identities

### PR DESCRIPTION
**Pull request by ioigoume**
Limit access to OrgIdentities for COU Administrators.

### CODE REVIEW

File: app/View/CoPeople/fields.inc [hunk 1]
```diff
@@ -1246,17 +1249,18 @@
                     </td>
                     <td>
                       <?php
-                        print $this->Html->link(_txt($e && !$es ? 'op.edit' : 'op.view'),
+                        print $this->Html->link(_txt($e && !$es && $org_e ? 'op.edit' : 'op.view'),
```
Comment (lionick) [inline app/View/CoPeople/fields.inc]: Why dont you use $permissions['edit'] instead of $org_e?

**Review by lionick — CHANGES_REQUESTED** [review]

File: app/View/CoPeople/fields.inc (hunk 1 above)
Comment (ioigoume) [reply app/View/CoPeople/fields.inc]: Done

**Review by lionick — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**